src/SampleExtension/SampleExtension.csproj

```diff
@@ -81,9 +84,8 @@
       <HintPath>..\packages\Prism.4.1.0.0\lib\NET40\Microsoft.Practices.Prism.Interactivity.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="Microsoft.Practices.ServiceLocation, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\CommonServiceLocator.1.0\lib\NET35\Microsoft.Practices.ServiceLocation.dll</HintPath>
-      <Private>False</Private>
+    <Reference Include="Microsoft.Practices.ServiceLocation, Version=1.3.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">

```

Comment:
this seems to have switched from `private` false to the default (true) - but what is this for anyway?

Reply:
I think that's related to `Prism`, if I recall correctly. I only consolidated every project to use the same version of it.

I don't think it actually matters whether this is private or not because it's not copied to the output folders in `dynamo_package` or `dynamo_viewExtension`.